pkgs/data/fonts/newcomputermodern/default.nix

```
@@ -0,0 +1,28 @@
+{ lib, stdenvNoCC, fetchzip, texlive }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "newcomputermodern";
+  version = "4.4";
```

```suggestion
  inherit (src) version;
```
Avoiding drifts of the version number and the actual version is probably also a good idea. (This also applies to iwona).

---

Fixed!